crates/locutus-node/src/errors.rs

```diff
@@ -1,34 +1,79 @@
-use warp::{hyper::StatusCode, reject::Reject, reply, Rejection, Reply};
+use axum::http::StatusCode;
+use axum::response::{Html, IntoResponse, Response};
+use locutus_stdlib::client_api::ErrorKind;
+use std::fmt::{Display, Formatter};
 
-use super::*;
+#[derive(Debug)]
+pub enum Error {
```

Comment:
does this need o be public really? is enough with it being pub(crate) ?

Reply:
also we probably should use a more descriptive name for this, like `WebSocketApiError`

Reply:
The problem is that it is used outside of this crate